region/maridia/inner-yellow/Plasma Spark Room.json

```diff
@@ -953,6 +1127,45 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [4, 5],
+      "name": "Speedy Jump",
+      "entranceCondition": {
+        "comeInRunning": {
+          "minTiles": 21,
+          "speedBooster": true
+        }
+      },
+      "requires": [
+        "canInsaneJump"
+      ],
+      "note": [
+        "Jump on the last possible frame, and get around the ledge while remaining in spin."
```

Comment:
True. I had tested it at the minimal speed (using 21 tiles) and it didn't help. But at high enough speeds it does help. Added a note about it.

We could split the lenient version off into a separate strat (with longer runway requirement) if you want? I'm guessing even the lenient version would still fit in "Extreme" because of the high risk of failure requiring a reset?

Reply:
I think they are still similar enough to call the same strat.